stmhal/boards/STM32F429DISC/pins.csv

```diff
@@ -0,0 +1,38 @@
+SW,PA0
+PA0,PA0
+PA5,PA5
+PA8,PA8
+PA9,PA9
+PA10,PA10
+PA13,PA13
+PA14,PA14
```

Comment:
Ok I added all pins except PB14/PB15 which are used for USB_HS and on my board the solder bridge is not fitted. Therefore this pin is not accessible.